### PR TITLE
Apply max quota to users and spaces 

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/8490

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-admin-settings/src/components/Spaces/ContextActions.vue [hunk 1]
```diff
@@ -34,6 +36,8 @@ export default defineComponent({
     }
   },
   setup(props) {
+    const store = useStore()
+    const maxQuota = computed(() => store.getters.capabilities.spaces?.max_quota)
```
Comment (JammingBen) [inline packages/web-app-admin-settings/src/components/Spaces/ContextActions.vue:40]: Please introduce a new composable in `useCapability.ts` for this (see examples there).

File: packages/web-pkg/src/components/QuotaSelect.vue [hunk 2]
```diff
@@ -142,10 +137,10 @@ export default {
           value,
           displayValue,
           displayUnit: 'GB',
-          error: this.$gettextInterpolate(
-            this.$gettext('Max quota limit of %{ limit } GB exceeded'),
-            { limit: this.maxQuota * Math.pow(10, -9) }
-          ),
+          error: this.$gettext('Please enter a value less than %{ maxQuota } GB', {
```
Comment (JammingBen) [inline packages/web-pkg/src/components/QuotaSelect.vue:140]: It should be `...less or equal than...`, no?

**Review by JammingBen — COMMENTED** [review]
+Changelog, then LGTM.

File: packages/web-pkg/src/components/QuotaSelect.vue (hunk 2 above)
Comment (JanAckermann) [reply packages/web-pkg/src/components/QuotaSelect.vue:140]: yes, wrong in the ticket

**Review by JammingBen — APPROVED** [review]

**Pull request merged**